.changeset/soft-impalas-remember.md

```diff
@@ -0,0 +1,7 @@
+---
+'@keystone-next/app-basic': patch
+'@keystone-next/auth': patch
+'@keystone-next/types': major
```

Comment:
Save argument as in #4427, correct usage of APIs isn't going to cause a problem with the updated types, imo this should be patch.

Otherwise almost every change to the types package is going to result in a major.